api/constants/constants.go

```diff
@@ -134,6 +134,9 @@ const (
 
 	// SSHRSAType is the string which specifies an "ssh-rsa" formatted keypair
 	SSHRSAType = "ssh-rsa"
+
+	// LicenseExpiredType is defines a license expired notification
+	LicenseExpiredType = "licenseExpired"
```

Comment:
```suggestion
	LicenseExpiredNotification = "licenseExpired"
```

I'm not a fan of using this `constants` package unless you have something that really needs to be accessed in many places throughout the app.

Since licensing is a function of the auth server only, putting this in `lib/auth` or maybe even `api` might make more sense. Up to you.

Reply:
That makes sense. For now we only need to access the license notifications in the `/lib/auth` package so we can localize it there.